CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
```

Comment:
Thanks for the feeback.

The `level={1}` is not required since the default level value is 1. But if you want to have a different level style you have to add it manually.

Reply:
And yes, I need it for the SideNavigation on the responsive app.
Main items have a link and should display sub items when clicking on the arrow.